raiden/storage/sqlite.py

```diff
@@ -51,21 +76,38 @@ class FilteredDBQuery(NamedTuple):
     inner_operator: Operator
 
 
+class EventEncodedRecord(NamedTuple):
+    event_identifier: EventID
```

Comment:
I did not consolidated the three types because their attributes are different.

Reply:
I would still like to see the record primary identifier field names be in unison with the database layout: see https://github.com/raiden-network/raiden/issues/4078